src/clientSideScene/sceneEntities.ts

```diff
@@ -1465,6 +1465,23 @@ export class SceneEntities {
         to,
       })
   }
+  /**
+   * Update the base color of each of the THREEjs meshes
+   * that represent each of the sketch segments, to get the
+   * latest value from `sceneInfra._theme`
+   */
+  updateSegmentBaseColor(newColor: Themes.Light | Themes.Dark) {
+    const newColorThreeJs = getThemeColorForThreeJs(newColor)
+    Object.values(this.activeSegments).forEach((group) => {
+      group.userData.baseColor = newColorThreeJs
+      group.traverse((child) => {
+        if (child instanceof Mesh) {
+          const mat = child.material as MeshBasicMaterial
```

Comment:
Is there no way to guarantee this is a MeshBasicMaterial? To avoid `as` ?

Reply:
I bet there is, good call